### PR TITLE
Exclude input processors for xcube gen into seperate repositories

### LINKED ISSUE #49: Move specific `xcube gen` input processors into separate repos
`xcube gen` input processor implementations are often very specific with respect to the supported data input format. Some of them require physical datasets for testing to be included in the sources. `xcube` on the other hand provides a generic API and CLI and does not rely on physical test datasets. 

Input processors are also developed by different development teams and should therefore have different repositories with own responsibilities and issues.

Move specific input processors should be moved into separate repositories.

### PR DESCRIPTION
**Pull request by AliceBalfanz**
This PR deletes the input processors from xcube gen and they can only be used via xcube-gen- repositories within the DCS4COP organization. This allows development of the input processors appart from the source code of xcube. This is strongly connected to following pull requests: 

- https://github.com/dcs4cop/xcube-gen-rbins/pull/1
- https://github.com/dcs4cop/xcube-gen-vito/pull/1
- https://github.com/dcs4cop/xcube-gen-bc/pull/1

closes #49

### CODE REVIEW

**Comment by forman** [comment]
@AliceBalfanz great! Lets complete review together tomorrow...

File: CHANGES.md [hunk 1]
```diff
@@ -25,6 +25,7 @@
 * Included possibility to read the input file paths from a text file. (#47)
 * Restructured and clarified code base (#27)
 * Moved to Python 3.7 (#25)
+* Excluding all input processors except for the default one. They are now own repositories within the DCS4COP organisation. (#49)
```
Comment (forman) [inline CHANGES.md]: Mention *as plugins*

File: README.md [hunk 2]
```diff
@@ -187,14 +187,14 @@ Is used to generate data cubes.
...
 
     Options:
       -p, --proc INPUT_PROCESSOR      Input processor type name. The choices as
-                                      input processor are: [].  Additional
-                                      information about input processors can be
-                                      accessed by calling xcube generate_cube
-                                      --info
+                                      input processor and additional information
+                                      about input processors  can be accessed by
+                                      calling xcube gen --info .
```
Comment (forman) [inline README.md]: Add `defaults to "default" - the default input processor that can deal with most common datasets conforming with the CF conventions.`

File: README.md [hunk 3]
```diff
@@ -238,11 +238,8 @@ Is used to generate data cubes.
     $ xcube gen --info
     input processors to be used with option --proc:
       default                           Single-scene NetCDF/CF inputs in xcube standard format
-      rbins-seviri-highroc-scene-l2     RBINS SEVIRI HIGHROC single-scene Level-2 NetCDF inputs
-      rbins-seviri-highroc-daily-l2     RBINS SEVIRI HIGHROC daily Level-2 NetCDF inputs
-      snap-olci-highroc-l2              SNAP Sentinel-3 OLCI HIGHROC Level-2 NetCDF inputs
-      snap-olci-cyanoalert-l2           SNAP Sentinel-3 OLCI CyanoAlert Level-2 NetCDF inputs
-      vito-s2plus-l2                    VITO Sentinel-2 Plus Level 2 NetCDF inputs
+
```
Comment (forman) [inline README.md]: Optional: Leave here, and say that a 5 input processor have been installed via plugins.

File: setup.py [hunk 4]
```diff
@@ -77,9 +77,6 @@
         ],
         'xcube_plugins': [
```
Comment (forman) [inline setup.py]:
```suggestion
        # This is xcube's extension point for new plugins. Use this extension point to add your own
        # xcube plugin in the setup.py of your Python package sources.
        'xcube_plugins': [
```

File: xcube/cli/gen.py [hunk 5]
```diff
@@ -135,6 +134,8 @@ def _format_info():
 
     help_text = '\ninput processors to be used with option --proc:\n'
     help_text += _format_input_processors(input_processors)
+    help_text += '\nFor more input processors use existing "xcube-gen-..." plugins ' \
+                 'from the github organisation DCS4COP or write own plugin.\n'
```
Comment (forman) [inline xcube/cli/gen.py]:
```suggestion
                 "from xcube's GitHub organisation or write own plugin.\n"
```

**Comment by AliceBalfanz** [comment]
@forman Please take another look. I included requested changes.

File: xcube/cli/gen.py (hunk 5 above)
Comment (dzelge) [reply xcube/cli/gen.py]: from the github organisation DCS4COP or write your own plugin.

Comment (dzelge) [reply xcube/cli/gen.py]: maybe link to teh doczumentation how to write your own plugin?

**Pull request merged**